vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -398,15 +398,52 @@ import { ItemCache } from '@vaadin/vaadin-grid/src/vaadin-grid-data-provider-mix
       })
 
       const sorterChangeListener = tryCatchWrapper(function(_, oldValue) {
+        let isSortersDuplicate = false;
+        let sortersDuplicate = [];
+        console.log(grid._sorters)
+        if(grid._sorters.length > 1) {
+          for(let i = 0; i < grid._sorters.length - 1; i++) {
+            for(let j = i + 1; j < grid._sorters.length; j++) {
+              if(grid._sorters[i].direction === grid._sorters[j].direction &&
+                  grid._sorters[i].path === grid._sorters[j].path) {
+                sortersDuplicate.push(j);
+                isSortersDuplicate =true;
+                break;
+              }
+            }
+            if(isSortersDuplicate) {
+              break;
+            }
+          }
+        }
+        if(sortersDuplicate.length > 0 ) {
+          let tempGrid = grid._sorters;
```

Comment:
`tempGrid` as a variable name for for `grid._sorters`?

Reply:
changed name to tempSorters to make more sense.